### ISSUE TITLE
error 500 nginx

### ISSUE THREAD

**Issue by patrick-mota** [etopian/docker-koel#6]
I guess something is wrong.
When I launch your project with a command like this one : 
docker run -e DB_HOST="172.17.0.2" -e DB_DATABASE="koel" -e DB_DATABASE="koel" -e DB_USERNAME="root" -e DB_PASSWORD="root" -e ADMIN_EMAIL="mail@mail.com" -e ADMIN_NAME="admin"  -e ADMIN_PASSWORD="root" -e APP_DEBUG=false -e AP_ENV=production -v /my-mp3s:/DATA/music -d etopian/docker-koel 
then
docker exec berserk_dijkstra su nginx -c "cd /DATA/htdocs && php artisan init"

I got all working message but when I go to my container index after login I got : 

Whoops, looks like something went wrong.

Looking to phpfpm logs:
[22-Dec-2015 16:49:26] WARNING: [pool www] child 131 said into stderr: "ERROR: Unable to set php_admin_value 'php_value max_execution_time'"

and in nginx:
- \- - [22/Dec/2015:16:45:09 +0000]  "POST /login HTTP/1.1" 500 4532 "http://172.17.0.3/login" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:41.0) Gecko/20100101 Firefox/41.0"

(I'm not the only one I sent your repository to 5 different people, all got the same problem)

Seems some permission problem but I don't get it.
The container should work out of box.. have a look please ;)

**Comment by etopian** [comment]
thanks. looking into it, expect something by tomorrow at the latest.

**Comment by patrick-mota** [comment]
No problem, If you need some help feel free. Thank you

**Comment by doda-zz** [comment]
@patrick-mota Just ran into the same issue, there's some problem with the passthrough of the `DB_DATABASE`, so it's using the default `forge`. You should be able to get it running by setting `DB_DATABASE=forge`

**Comment by patrick-mota** [comment]
@doda nice catch ! Thank you, I can now launch koel app at least :) gonna try out a bit more.
If all works, I'll submit a fix for the docker image.

**Comment by Aidenir** [comment]
I'm seeing the same issue, but I don't understand what @doda means so I'm not sure how to fix this. Any help? I've gotten it to talk to the database since the tables are created in the koel db at least.
(I hate apps that requires more frameworks and shit to start than linux itself :( )

**Comment by doda-zz** [comment]
Set forge as the database name

**Comment by patrick-mota** [comment]
@Aidenir Yes just set forge as database name.... the problem is the app (laravel) doesn't take the ENV variable for the database so it will set the default one (forge)

**Comment by Aidenir** [comment]
Ok now I get it :P Thanks, that made it work.
